pkg/continuousprofiling/profiler.go

```diff
@@ -77,7 +100,7 @@ func validateServerAddress(address string) (string, error) {
 	// We default to https unless http is specified
 	sanitizedAddress := urlfmt.FormatURL(address, urlfmt.HTTPS, urlfmt.NoTrailingSlash)
 	if _, err := url.Parse(sanitizedAddress); err != nil {
-		return "", errors.Wrapf(err, "unable to parse the server address %q", address)
+		return "", errox.InvalidArgs.Newf("unable to parse server address %q", address).CausedByf(err.Error())
```

Comment:
Sorry, my bad, made a typo in the previous comment.
```suggestion
		return "", errox.InvalidArgs.Newf("unable to parse server address %q", address).CausedBy(err)
```